app/controllers/users_controller.rb

```diff
@@ -1,6 +1,6 @@
 class UsersController < ApplicationController
-  before_action :admin?, only: [:index, :show, :new, :create, :edit, :update, :destroy]
-  before_action :set_user, only: [:show, :destroy]
+  before_action :admin?, only: %i[index show new create edit update destroy]
+  before_action :set_user, only: %i[show edit update destroy]
```

Comment:
I think this is more a function of our editors' and rubocop setup than abalone's. Works either way, just not sure if we're injecting our code-style into their project.

Reply:
you're right @wadewinningham . I think I'll revert this to keep it consistent with the rest of the codebase.